desk/ted/vitals/connection-check.hoon

```diff
@@ -104,12 +110,15 @@
   ^-  update:vitals
   [target now %pending pending]
 ++  get-qos
-  |=  =ship
+  |=  peer=ship
   =/  m  (strand ,qos:ames)
   ^-  form:m
-  ;<    state=ship-state:ames
-      bind:m
-    (scry:io ship-state:ames ~[%ax %$ %peers (scot %p ship)])
+  ;<  peers=(map ship ?(%alien %known))  bind:m
+    (scry:io (map ship ?(%alien %known)) ~[%ax %$ %peers])
+  ?.  (~(has by peers) peer)
+    (pure:m [%dead *@da])
```

Comment:
Wouldn't `%unborn` be more accurate? (idk if that would get handled correctly further up the stack, but, it's a valid case for `$qos:ames`, so it should...)

Reply:
Yes, I think you're right. I'll update this in my next commit.